<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageDatabaseDialog</class>
 <widget class="QDialog" name="ManageDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>515</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>ManageCatalogs</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
  border: 1px solid gray;
  border-radius: 3px;
  padding: 5px;
}
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string extracomment="Name of the frame in which buttons for operating one catalog are arranged">Selected Catalog</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string extracomment="Button to add a new catalog">&amp;AddNew</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string extracomment="Button for updating the contents of registered catalog">UpdateDatabase</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string extracomment="Button to delete information of registered catalog">&amp;Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string extracomment="Button to edit information on registered catalog">&amp;Edit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string extracomment="Name of the frame in which buttons for operating all catalogs are arranged">All Catalogs</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="deleteAllButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">padding 5px;</string>
        </property>
        <property name="text">
         <string>DeleteAll</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="updateAllButton">
        <property name="text">
         <string extracomment="登録済みのカタログの内容を更新するボタン">UpdateAll</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="cancelButton">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
padding: 8px;
</string>
        </property>
        <property name="text">
         <string notr="true">Cancel work</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="volumeNameLabel">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManageDatabaseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManageDatabaseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageDatabaseDialog</receiver>
   <slot>onAddButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageDatabaseDialog</receiver>
   <slot>onDeleteButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageDatabaseDialog</receiver>
   <slot>onEditButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageDatabaseDialog</receiver>
   <slot>onUpdateButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteAllButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageDatabaseDialog</receiver>
   <slot>onDeleteAllButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateAllButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageDatabaseDialog</receiver>
   <slot>onUpdateAllButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ManageDatabaseDialog</receiver>
   <slot>onCancelButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddButton_clicked()</slot>
  <slot>onDeleteButton_clicked()</slot>
  <slot>onEditButton_clicked()</slot>
  <slot>onUpdateButton_clicked()</slot>
  <slot>onDeleteAllButton_clicked()</slot>
  <slot>onUpdateAllButton_clicked()</slot>
  <slot>onCancelButton_clicked()</slot>
 </slots>
</ui>
